env/env.go

```diff
@@ -164,6 +164,19 @@ func eachEnvFile(dir string, fn func(k, v string) error) error {
 		if f.IsDir() {
 			continue
 		}
+		if f.Mode()&os.ModeSymlink != 0 {
+			lnPath, err := filepath.EvalSymlinks(filepath.Join(dir, f.Name()))
+			if err != nil {
+				return err
+			}
+			lnFile, err := os.Stat(lnPath)
+			if err != nil {
+				return err
+			}
```

Comment:
```suggestion
			lnFile, err := os.Stat(filepath.Join(dir, f.Name()))
			if err != nil {
				return err
			}
```

`os.Stat` already stats the link target (`os.Lstat` would stat the link) so there is no need to evalulate the links first.